### ISSUE TITLE
Person and Relationships Citations omitted

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/CardView#230]
The Person CardView Citations panels omitted Event citations (where the Person was 'Primary' role and another where they were 'Witness' role), including Family Events where the Person was a Spouse.

The Relationships CardView omitted the same but also omitted other citations related to data all the People, Families and Events shown in the View mode.  It seems like the Relationship should list all Citations for any  object represented in the View.

A zipped tiny `_YodaCitations_2022-11-18.gramps` sample file with various Citations is attached.  This allows the Citation category View to be used as a sanity check.

[_GrampsCitationsTest.zip](https://github.com/cdhorn/CardView/files/10044686/_GrampsCitationsTest.zip)

**Comment by cdhorn** [comment]
Yes it could be extended to show all citations relevant to the person.  Something similar could be done for families. 

I find showing event citations for non-primary roles to be problematic.  In my mind the event reference should hold the citations regarding the role the person played in the event, not the event record itself as there is no way to know which citations on the event pertain to which participants. Hence the PR I submitted for consideration for 5.2 that added CitationBase to the EventRef object.
